app/models/collection.rb

```diff
@@ -55,4 +55,8 @@ def reviewers_and_managers
   def first_version?
     version == 1
   end
+
+  def to_param
+    druid
+  end
```

Comment:
The changes to the models in this commit are the key change that gets us back to being able to be more Rails-y in our path and URL helpers, e.g., `collection_path(collection)`, without losing the ability to craft special-case URLs such as the explicit "wait" paths, in which case we want `wait_collections_path(collection.id)`.